src/css/hero.css

```diff
@@ -1,159 +1,504 @@
-    height: 38px;
+@media only screen and (min-width: 1280px) {
+  .hero-wrapper {
+    position: absolute;
+    z-index: -1;
+    top: 336px;
+    left: 50%;
+    transform: translateX(-50%);
+    justify-content: space-between;
+    width: 1156px;
   }
 }
+
+.wrapper-left-block,
+.wrapper-right-block {
+  display: flex;
+  flex-direction: column;
+  gap: 44px;
+}
+
 @media screen and (min-width: 1280px) {
-  .hashtag h3 {
-    font-family: 'DM Sans', sans-serif;
-    font-weight: 500;
-    font-size: 12px;
-    line-height: 1.16667;
-    letter-spacing: -0.04em;
-    text-transform: uppercase;
-    text-align: center;
-    color: #fff;
-    text-shadow: 0 4px 70px 0 rgba(0, 0, 0, 0.1);
+  .left-block-description {
+    margin-top: 72px;
   }
 }
-@media screen and (min-width: 1280px) {
-  .hero-focus {
-    border-radius: 500px;
-    width: 452px;
-    height: 702px;
-    background: #fff;
+
+@media screen and (max-width: 767px) {
+  .left-block-card,
+  .right-block-card {
+    margin: 0 auto;
   }
 }
-@media screen and (min-width: 1280px) {
-  .hero-image {
-    width: 1053px;
-    height: 702px;
+
+@media only screen and (min-width: 768px) {
+  .left-block-card {
+    display: flex;
+    align-self: end;
+  }
+}
+
+.right-block-card {
+  order: 1;
+}
+
+@media only screen and (min-width: 768px) {
+  .right-block-card {
+    order: 0;
+    display: flex;
+    align-self: start;
   }
 }
+@media only screen and (min-width: 1280px) {
+  .right-block-card {
+    align-self: end;
+  }
+}
+.hero-card {
+  text-align: center;
+}
+
+.hero-card-img {
+  z-index: 2;
+}
+
+.hero-card-watermelon {
+  position: relative;
+  width: 249px;
+  min-height: 387px;
+  background-color: #a4cb66;
+  border-radius: 500px;
+}
+
+.hero-card-watermelon > .oval {
+  position: absolute;
+  top: 209px;
+  left: 50%;
+  transform: translateX(-50%);
+  z-index: 1;
+}
+
+.hero-card-strawberry {
+  position: relative;
+  width: 249px;
+  min-height: 387px;
+  background-color: #f5ce13;
+  border-radius: 500px;
+}
+
+.hero-card-strawberry > .oval-strawberries {
+  position: absolute;
+  top: 191px;
+  left: 50%;
+  transform: translateX(-50%);
+  z-index: 1;
+}
+
+.hero-card-title {
+  padding-top: 28px;
+  text-align: center;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 1.17;
+  letter-spacing: -0.04em;
+  color: #fff;
+}
+
+.title-card-watermelon {
+  margin-bottom: 61px;
+}
+
+.title-card-strawberry {
+  margin-bottom: 19px;
+}
+
+.hero-card-text {
+  margin-right: auto;
+  margin-left: auto;
+  text-transform: uppercase;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 1.17;
+  color: #fff;
+  text-align: center;
+  z-index: 3;
+}
+
+.text-card-watermelon {
+  max-width: 155px;
+}
+
+.text-card-strawberry {
+  padding-top: 15px;
+  max-width: 180px;
+}
+
+.title-item {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.hero-hashtags {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  column-gap: 8px;
```

Comment:
po prostu gap: 8px; i flex-direction chyba niepotrzebny.

Reply:
w tablecie są 2 kolumny i 2 wiersze jak daję tylko gap 8 to nie działa